web/templates/home_page/about.html.eex

```diff
@@ -0,0 +1,19 @@
+<header class="banner-body">
+  <h1 class="banner-title">
+    Building community and sharing&nbsp;ideas.
+  </h1>
+  <p>
+    Triangle Designers is a resource for designers throughout North Carolina's
+    Triangle to connect, collaborate, and find prospective career opportunities.
```

Comment:
Same here, I wonder if this can put into a constant. Maybe something like `Community.Constants` and it has functions like `.description` and `.name` and `.email`?